controls/autocomplete/display-text.md

```diff
@@ -8,14 +8,14 @@ slug: autocomplete-display-text-formatter
 
 # .NET MAUI Autocomplete Display Text Formatting
 
-The AutoComplete control provides the option to format the visualized text in the input, so you could modify the displayed details when item is get from the suggestion view. The format of the text could be defined when `AutoComplete` `DisplayMode` is `Plain` or `Tokens`:
+The AutoComplete control provides the option to format the visualized text in the input, so you can modify the displayed details when item is get from the suggestion view. The format of the text can be defined when `AutoComplete` `DisplayMode` is `Plain` or `Tokens`:
 
 * `DisplayTextFormatter`(IDisplayTextFormatter): Defines the formatter of the selected item.
 
-There are two options to define the formatter of the selected item:
+Two options to define the formatter of the selected item:
```

Comment:
Be careful when removing "There are", "There is", etc because your sentence may lose its formal subject and verb. When removing the phrase, also make sure that you rewrite the sentence so it is grammatically correct.
```suggestion
To define the formatter of the selected item, you can use the following options:
```